MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1052,7 +1052,7 @@ private bool IsClose(double d1, double d2, double variance = .005)
                 return null;
             }
 
-            return divisor == 0f ? null : dividend / divisor;
+            return divisor == 0f || dividend == 0f ? null : dividend / divisor;
```

Comment:
I think this one should return `0`, @cvium ?

Reply:
0/999999 is still 0, so that wouldn't change anything. I'm not sure if this is the place to fix it

Reply:
This is meant to fix the issue where there is logic: `1 / GetFrameRate(....) ?? 0` which turns into Inifnity when you go `1/0`

Reply:
You are right, the problem isn't here, I've made a PR that hopefully fixes the issue
https://github.com/jellyfin/jellyfin/pull/7120

Reply:
LGTM thanks